src/commissaire/containermgr/trivial.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2017  Red Hat, Inc
```

Comment:
How strongly do you feel about that?

Because it's imported as a plugin, it would have to be specified as a full module path in a `ContainerManagerConfig`.  And in order for that to work, I'd have to hack the `PYTHONPATH` in the `container-manager-service` environment, and that then involves hacking its `.service` file in the `Vagrantfile`.

In other words it's getting pretty ugly, unless I'm missing a simpler way.